### DISCUSSION TITLE
Local development with monaco-vscode-api

### DISCUSSION THREAD

**Discussion by dlvhdr** [TypeFox/monaco-languageclient#407] [General]
I see you're using aliasing `vscode` to `monaco-vscode-api` in https://github.com/TypeFox/monaco-languageclient/blob/fbc2f1e314264f6aa092ed35faeb7cb31131aa6b/packages/client/package.json#L52

How can I override this and provide my own build of `monaco-vscode-api`?
I didn't manage to do it by modifying the `package.json` so I had to manually run `ln -s` to create the link in the `node_modules` which is not ideal.
Is there a better way to do this?

@CGNonofr

**Comment by CGNonofr** [comment]
I didn't find any way to do it either, so it's a pain to test it in local, I usually just `cp` the dist folder... :shrug: I'm interested if you find any solution
